frontend/src/Components/Cards.jsx

```diff
@@ -56,7 +56,7 @@ const Cards = ({ dish, setRecipes, recipes, index }) => {
 
         setRecipes(updatedRecipes);
         await navigator.clipboard.writeText(postURL);
-        toast.success("Copied!");
+        toast.success("URL copied to clipboard successfully");
       } else {
```

Comment:
@mehul-m-prajapati  Keep the text as it was before!

Reply:
@AlfiyaSiddique : okay updated it